packages/office-ui-fabric-react/src/components/Tooltip/TooltipHost.tsx

```diff
@@ -48,10 +50,12 @@ export class TooltipHost extends BaseComponent<ITooltipHostProps, ITooltipHostSt
         { ...{ onBlurCapture: this._onTooltipMouseLeave } }
         onMouseEnter={ this._onTooltipMouseEnter }
         onMouseLeave={ this._onTooltipMouseLeave }
+        aria-describedby={ tooltipId }
       >
         { children }
         { isTooltipVisible && (
           <Tooltip
+            id={ tooltipId }
```

Comment:
@ostreifel the tooltip is lying to us :) these props are never applied, unfortunately